### PR TITLE
code review suggestions

### PR DESCRIPTION
**Pull request by corinmanning2315**
**Description/Analysis of the Project**
This is an app that helps you find recycling centers and save your favorite centers.

**Was the program available in UC Github on time?**
Yes

**Is the program documented/commented well enough for you to understand?**
Yes

**Does the program compile?**
Yes

**Rationale Behind Changes**
I just changed a few things so that the project met best practices standards. There were no noticeable errors, I just fixed some warnings.

**3 Commits**
1. Added logo
ahlersml/MyCleaningLog@287baa8
2. Fixed errors
ahlersml/MyCleaningLog@bfd444a
3. Updated color palette
ahlersml/MyCleaningLog@89ec9c4

**Things I Learned**
1.  How to do an inline variable
2. Not having functions in camelcase is not a big deal - the code still runs
3. Kotlin best practices of structural equality

### CODE REVIEW

File: app/src/main/java/com/findmyrecycling/service/ProductService.kt [hunk 1]
```diff
@@ -13,9 +13,8 @@ class ProductService{
     suspend fun fetchProducts() : List<Product>? {
         return withContext(Dispatchers.IO) {
             val service = RetrofitClientInstance.retrofitInstance?.create(IProductDAO::class.java)
-            val products = async {service?.getAllProducts()}
-            var result = products.await()?.awaitResponse()?.body()
-            return@withContext result
+            val products = async { service?.getAllProducts() }
+            return@withContext products.await()?.awaitResponse()?.body()
```
Comment (corinmanning2315) [inline app/src/main/java/com/findmyrecycling/service/ProductService.kt:17]: updated variable to inline

File: app/src/main/java/com/findmyrecycling/service/ProductService.kt [hunk 2]
```diff
@@ -13,9 +13,8 @@ class ProductService{
     suspend fun fetchProducts() : List<Product>? {
         return withContext(Dispatchers.IO) {
             val service = RetrofitClientInstance.retrofitInstance?.create(IProductDAO::class.java)
-            val products = async {service?.getAllProducts()}
-            var result = products.await()?.awaitResponse()?.body()
-            return@withContext result
+            val products = async { service?.getAllProducts() }
```
Comment (corinmanning2315) [inline app/src/main/java/com/findmyrecycling/service/ProductService.kt:16]: edited spacing

File: app/src/test/java/com/findmyrecycling/ProductTests.kt [hunk 3]
```diff
@@ -39,7 +39,7 @@ class ProductTests {
     }
 
     @After
-    fun TearDown(){
+    fun tearDown(){
```
Comment (corinmanning2315) [inline app/src/test/java/com/findmyrecycling/ProductTests.kt:42]: fixed camelcase

File: app/src/test/java/com/findmyrecycling/ProductTests.kt [hunk 4]
```diff
@@ -76,7 +76,7 @@ class ProductTests {
         assertTrue(allProducts!!.isNotEmpty())
         var containsCellPhones = false
         allProducts!!.forEach {
-            if (it.product.equals("Cell Phones")) {
+            if (it.product == "Cell Phones") {
```
Comment (corinmanning2315) [inline app/src/test/java/com/findmyrecycling/ProductTests.kt:79]: simplified operator

File: app/src/test/java/com/findmyrecycling/ProductTests.kt [hunk 5]
```diff
@@ -88,7 +88,7 @@ class ProductTests {
         assertTrue(allProducts!!.isNotEmpty())
         var containsBatteries = false
         allProducts!!.forEach {
-            if (it.product.equals("Rechargable Batteries") || it.product.equals("Lead Acid Batteries")) {
+            if (it.product == "Rechargable Batteries" || it.product.equals("Lead Acid Batteries")) {
```
Comment (corinmanning2315) [inline app/src/test/java/com/findmyrecycling/ProductTests.kt:91]: simplified operator

File: app/src/test/java/com/findmyrecycling/ProductTests.kt [hunk 6]
```diff
@@ -100,7 +100,7 @@ class ProductTests {
         assertTrue(allProducts!!.isNotEmpty())
         var containsNothing = false
         allProducts!!.forEach {
-            if (!it.product.equals("Human Remains")) {
+            if (it.product != "Human Remains") {
```
Comment (corinmanning2315) [inline app/src/test/java/com/findmyrecycling/ProductTests.kt:103]: simplified operator

File: app/src/test/java/com/findmyrecycling/ProductTests.kt [hunk 7]
```diff
@@ -147,7 +147,7 @@ class ProductTests {
         assertTrue(allProducts!!.isNotEmpty())
         var containsCellPhone = false
         allProducts!!.forEach {
-            if (it.product.equals("Cell Phone")) {
+            if (it.product == "Cell Phone") {
```
Comment (corinmanning2315) [inline app/src/test/java/com/findmyrecycling/ProductTests.kt:150]: simplified operator

File: app/src/main/java/com/findmyrecycling/service/ProductService.kt (hunk 1 above)
Comment (discospiff) [reply app/src/main/java/com/findmyrecycling/service/ProductService.kt:17]: I caution against doing too much in one line.  If you get an exception on this line, it's hard to determine what went wrong.

File: app/src/test/java/com/findmyrecycling/ProductTests.kt (hunk 3 above)
Comment (discospiff) [reply app/src/test/java/com/findmyrecycling/ProductTests.kt:42]: Good.

File: app/src/test/java/com/findmyrecycling/ProductTests.kt (hunk 4 above)
Comment (discospiff) [reply app/src/test/java/com/findmyrecycling/ProductTests.kt:79]: This was covered in a previously submitted review. When that occurs, first in wins.

File: app/src/test/java/com/findmyrecycling/ProductTests.kt (hunk 5 above)
Comment (discospiff) [reply app/src/test/java/com/findmyrecycling/ProductTests.kt:91]: This was covered in a previously submitted review. When that occurs, first in wins.

File: app/src/test/java/com/findmyrecycling/ProductTests.kt (hunk 6 above)
Comment (discospiff) [reply app/src/test/java/com/findmyrecycling/ProductTests.kt:103]: This was covered in a previously submitted review. When that occurs, first in wins.

File: app/src/test/java/com/findmyrecycling/ProductTests.kt (hunk 7 above)
Comment (discospiff) [reply app/src/test/java/com/findmyrecycling/ProductTests.kt:150]: This was covered in a previously submitted review. When that occurs, first in wins.

**Comment by discospiff** [comment]
Many of these suggestions were covered in a previous review.  So, no need to merge this branch.

**Pull request merged**